src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1369,12 +1369,17 @@ function tryRemoveElement(elem) {
                             document.body.classList.add('hide-scroll');
                         }
 
-                        // don't animate on smart tv's, too slow
-                        if (options.fullscreen && browser.supportsCssAnimation() && !browser.slow) {
-                            return zoomIn(dlg).then(function () {
-                                return videoElement;
-                            });
-                        } else {
+                        if (options.fullscreen) {
+                            if (browser.tv && Screenfull.isEnabled) {
+                                Screenfull.request();
```

Comment:
Should there be a return here or maybe the next condition should be an `else if`? Or is it intended for TVs to continue through the rest of the following logic?

Reply:
Yes, if im correct tv's have `browser.slow` set to true anyway but its more clear to just immediately return